docs/rfcs/UserProfilesv2_schema.json

```diff
@@ -218,130 +286,44 @@
-                    "$ref": "#/definitions/Auth0"
-                },
-                "hris": {
-                    "$ref": "#/definitions/Auth0"
-                },
-                "auth0": {
-                    "$ref": "#/definitions/Auth0"
-                }
-            },
-            "title": "AccessInformation"
-        },
-        "Auth0": {
-            "type": "object",
-            "additionalProperties": false,
-            "properties": {
-                "metadata": {
-                    "$ref": "#/definitions/Metadata"
-                },
-                "value": {
-                    "type": "array",
-                    "items": {
-                        "$ref": "#/definitions/Auth0Value"
-                    }
-                }
-            },
-            "required": [
-                "metadata",
-                "value"
+        "Classification": {
+            "type": "string",
+            "enum": [
+                "PUBLIC",
+                "MOZILLA CONFIDENTIAL",
+                "WORKGROUP CONFIDENTIAL: STAFF ONLY",
+                "WORKGROUP CONFIDENTIAL",
+                "INDIVIDUAL CONFIDENTIAL"
             ],
-            "title": "Auth0"
+            "title": "Classification"
         },
-        "Auth0Value": {
-            "type": "object",
-            "additionalProperties": false,
-            "properties": {
-                "created": {
-                    "type": "string",
-                    "format": "date-time"
-                },
-                "lastUsed": {
-                    "type": "string",
-                    "format": "date-time"
-                },
-                "name": {
-                    "type": "string"
-                },
-                "value": {
-                    "oneOf": [
-                        {
-                            "type": "null"
-                        },
-                        {
-                            "type": "string"
-                        }
-                    ]
-                }
-            },
-            "required": [
-                "name",
-                "value"
+        "Publisher": {
+            "type": "string",
+            "enum": [
+                "auth0",
+                "mozilliansorg",
+                "ldap",
+                "hris"
             ],
-            "title": "Auth0Value"
+            "title": "Publisher"
         },
-        "ValueUnion": {
-            "oneOf": [
-                {
-                    "type": "array",
-                    "items": {
-                        "$ref": "#/definitions/ValueValue"
-                    }
-                },
-                {
-                    "type": "null"
-                },
-                {
-                    "type": "string"
-                }
-            ],
-            "title": "ValueUnion"
+        "Name": {
+            "type": "string",
+            "title": "Name"
         },
-        "Signature": {
+        "Type": {
```

Comment:
Is there any value in clarifying this schema type with instead of `Type` something like `SignatureType`?

Reply:
I second the disambiguation on Type here.  Since it's a reserved work in a lot of cases.  Better to be explicit.